tests/tests_integration/test_api/test_files.py

```diff
@@ -209,8 +211,37 @@ def test_upload_bytes_with_nordic_characters(self, cognite_client: CogniteClient
         external_id = "test_upload_bytes_with_nordic_characters"
 
         _ = cognite_client.files.upload_bytes(
-            content=content, name="nordic_chars.txt", external_id=external_id, overwrite=True
+            content=content,
+            name="nordic_chars.txt",
+            external_id=external_id,
+            overwrite=True,
         )
 
         retrieved_content = cognite_client.files.download_bytes(external_id=external_id)
         assert retrieved_content == content.encode("utf-8")
+
+    def test_upload_multipart(self, cognite_client: CogniteClient) -> None:
+        # Min file chunk size is 5MiB
+        content_1 = "abcde" * 1_200_000
+        content_2 = "fghij"
+
+        external_id = "test_upload_multipart"
+
+        cognite_client.config.file_transfer_timeout = 10
```

Comment:
Transferring 5 * 1.2 = 6 MiB may perhaps take more than 10 seconds in some cases ?
Should the timeout limit be increased to avoid flaky tests?

Reply:
Also, need to use `monkeypatch.setattr` to avoid side effects on other tests

Reply:
I just removed the change, it was a leftover from some weird issues we were having with this test.